pkgs/development/tcl-modules/critcl/default.nix

```diff
@@ -1,10 +1,11 @@
 { lib
```

Comment:
Why is this not in the `by-name` for tcl?

Reply:
Because the `callPackage` call has custom arguments, because of a dependency cycle between `critcl` and `tcllib`:

```nix
critcl = self.callPackage ../development/tcl-modules/critcl {
      tcllib = self.tcllib.override { withCritcl = false; };
    };
```

Should I move the override into `package.nix`?

Reply:
Oh gotcha, no it's fine ig.